tests/checkpoint/test_checkpoint_result_format.py

```diff
@@ -131,44 +144,28 @@ def reference_sql_checkpoint_config_for_multi_column_sum_table(
 
 
 @pytest.fixture()
-def expectation_config_expect_multicolumn_sum_to_equal() -> ExpectationConfiguration:
-    return ExpectationConfiguration(
-        expectation_type="expect_multicolumn_sum_to_equal",
-        kwargs={"column_list": ["a", "b", "c"], "sum_total": 30},
-    )
+def expect_multicolumn_sum_to_equal() -> Expectation:
+    return ExpectMulticolumnSumToEqual(column_list=["a", "b", "c"], sum_total=30)
```

Comment:
Changed all of these fixtures from configs to rich types so we use the Pydantic coercion as intended

Reply:
Seems great. I mostly skimmed the rest of this file because it looks like a pretty standard find/replace.